src/components/KToaster/ToastManager.ts

```diff
@@ -51,7 +51,7 @@ export default class ToastManager {
     // @ts-ignore
     const { key, timeoutMilliseconds, appearance, message, title } = args
 
-    const toastKey: any = key || useUniqueId()
+    const toastKey: any = key || getUniqueStringId()
```

Comment:
nit: should this be typed as `any`?

Reply:
Oh, not sure why it was `any`, fixed now